src/frontend/staff-portal/src/app/components/contact-info/contact-info.component.ts

```diff
@@ -71,7 +71,7 @@ export class ContactInfoComponent implements OnInit {
       rejectedReason: [null, Validators.maxLength(256)], // Optional
       country: [null, [Validators.required]],
       postalCode: [null, [Validators.required, Validators.maxLength(6), Validators.minLength(6)]],
-      driversLicenceNumber: [null, [Validators.required, Validators.maxLength(20)]],
+      driversLicenceNumber: [null, [Validators.required, Validators.minLength(7), Validators.maxLength(9)]],
```

Comment:
same as citizen portal

Reply:
as soon as dispute is retrieved validators are reset